README.md

```diff
@@ -64,11 +50,33 @@ The next file is `~/.gu/security-hq.local.conf`, which should be given by one of
 
 Another file you need will need (here again ask for a team member) is `~/.gu/security-hq-service-account-cert.json`.
 
+### Adding additional AWS accounts for local development
+
+As well as the `security-test` and `security` accounts, you can add additional AWS accounts to your local security-hq. This may be helpful for debugging purposes for example. You will need janus credentials for the account you want to add.
```

Comment:
(minor) If we are mentioning Janus, we should capitalize the J. Although, another option is to avoid mentioning Janus entirely. Some reports require IAM user credentials (which is why we create a security-test user) rather than Janus credentials.

```suggestion
When running security HQ locally, you can modify the list of AWS accounts to include additional account. For example, you may want to add a specific account for debugging purposes. You will need valid AWS credentials for any accounts you wish to include.
```